CHANGELOG.latest.md

```diff
@@ -1,4 +1,58 @@
-- Updates log message for `createAlias` to improve clarity
-    https://github.com/RevenueCat/purchases-ios/pull/498
-- Adds `rc_` to all Foundation extensions to prevent name collisions
-    https://github.com/RevenueCat/purchases-ios/pull/500
+
+### Identity V3:
```

Comment:
this was a tough list to compile. We usually make releases much more often, and we should continue to do so.